### PR TITLE
buggered git

### PR DESCRIPTION
**Pull request by jonathangoulding**

### CODE REVIEW

File: packages/forms-web-app/src/controllers/examination/session/examination-session.js [hunk 1]
```diff
@@ -50,6 +50,16 @@ const getExaminationSubmissionId = (session) => {
 
 const getExaminationEmailAddress = (session) => getExaminationSession(session).email;
 
+const getExaminationSubmissionRetryErrorCount = (session) => {
+	const examinationSession = getExaminationSession(session);
+	return examinationSession.retryErrorCount;
+};
+const setExaminationSubmissionRetryErrorCount = (session) => {
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/controllers/examination/session/examination-session.js:57]: Just a question around the naming as it seems to be doing 2 things, creating or incrementing it. Maybe a chance to split it out?

File: packages/forms-web-app/src/controllers/examination/submission-error/utils/get-page-data.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const { getProjectEmailAddress } = require('../../../../controllers/session/app-data-session');
+
+const { getExaminationSubmitCommentsURL } = require('./get-examination-submit-comments-url');
+
+const getPageData = (session) => {
+	const pageData = {
+		id: 'examination-submission-error',
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/controllers/examination/submission-error/utils/get-page-data.js:7]: Does this make sense to go into the config to be consistent?

File: packages/forms-web-app/src/controllers/examination/session/examination-session.js (hunk 1 above)
Comment (jonathangoulding) [reply packages/forms-web-app/src/controllers/examination/session/examination-session.js:58]: I think it's a setter. Consistent with other places we set defaults.

**Review by christopher-semmens — DISMISSED** [review]

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**